pkg/theme/text.go

```diff
@@ -0,0 +1,395 @@
+package theme
+
+import (
+	"github.com/gookit/color"
+)
+
+var (
```

Comment:
This file mixes a few responsibilities that I think should be split out:
1) defining the TextStyle interface
2) defining the two implementations of the TextStyle interface
3) exporting text styles that are available for use in the app

I think points 1 and 2 should be in a separate package within the `gui` package called `color` (or to avoid clashing with gookit we could call it 'style'. And we could have three files there: one for the interface, and one for each implementation.

Then we can leave this file with the exports of available styles.

What do you think?

Reply:
That would make it much more readable, i'll fix it